### PR TITLE
[fix bug 1314461] Web Compatibility Campaign page

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- WIP campaign page for Web Compat / Sauce Labs partnership.

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1314461

## Checklist
- [ ] Requires l10n changes.
- [ ] Related functional & integration tests passing.

### CODE REVIEW

**Comment by alexgibson** [comment]
There still may be some minor text / CSS changes, but I think this should be ready enough for review.

**Comment by alexgibson** [comment]
This PR is now complete in terms of copy/design. The Sauce Labs URL is still 404 currently, so we can't merge this until their page is live (scheduled for Dec 1st).

File: media/css/pebbles/components/_dev-newsletter.scss [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+    .content {
+        padding: 40px 0;
+        width: $width-phone;
+    }
+
+    // container for <h3> and <h4>
+    .form-title {
+        background: url('/media/img/newsletter/dev-newsletter.svg') center top no-repeat;
+        @include background-size(100px 100px);
+        padding: 120px 0 20px 0px;
```
Comment (jpetto) [inline media/css/pebbles/components/_dev-newsletter.scss]: Nit - no need for final `0px`.

**Comment by alexgibson** [comment]
> As they are almost identical, do you think it makes sense to have _dev-newsletter.scss import and override styles in _newsletter.scss rather than repeating rules?

Fair suggestion 👍

**Comment by alexgibson** [comment]
@jpetto thanks for the review, updated

**Comment by jpetto** [comment]
r+. Merge when ready. 🚦

**Comment by alexgibson** [comment]
@jpetto - some last minute copy changes from legal. Pretty straight forward but another look before merge would be appreciated?

**Comment by alexgibson** [comment]
Also we still need to hold off merging this until given the green light, just to be clear.

**Comment by alexgibson** [comment]
Ok we have the go-ahead to merge this when ready.

**Comment by jpetto** [comment]
Changes look good. r+ status remains in effect.

**Comment by alexgibson** [comment]
@jpetto squashed!

**Pull request merged**